src/EntryPoints/MediaWikiHooks.php

```diff
@@ -137,4 +138,21 @@ public static function onAlternateEdit( EditPage $editPage ): void {
 		}
 	}
 
+	public static function onArticleRevisionViewCustom(
+		RevisionRecord $revision,
+		Title $title,
+		int $oldId,
+		OutputPage $output
+	): bool {
+		if ( WikibaseRdfExtension::getInstance()->isConfigTitle( $title ) ) {
+			$lookup = WikibaseRdfExtension::getInstance()->newMappingPredicatesLookup();
```

Comment:
If we don't want the combined mappings, then we need to add the `WikiMappingPredicatesLookup` to the factory and get it here.